source/API/CMakeLists.txt

```diff
@@ -170,9 +170,10 @@ if(NOT LLDB_BUILT_STANDALONE)
   set(clang_headers_target symlink_clang_headers)
 endif()
 
+# Copy the clang resource directory.
 add_custom_command_target(
     unused_var
-    COMMAND "${CMAKE_COMMAND}" "-E" "copy_directory" "${CLANG_RESOURCE_PATH}" "${lib_dir}/lldb/clang"
+    COMMAND "${CMAKE_COMMAND}" "-E" "copy_directory" "${CLANG_RESOURCE_PATH}" "${lib_dir}/lldb/clang/${LLVM_PACKAGE_VERSION}"
     OUTPUT "${lib_dir}/lldb/clang"
```

Comment:
Do you need to update `OUTPUT` as well?

Reply:
I really don't understand CMake very well, but my thinking was that "${lib_dir}/lldb/clang" implied "${lib_dir}/lldb/clang/${LLVM_PACKAGE_VERSION}".

Reply:
I think that since `${LLVM_PACKAGE_VERSION}` is the only subdirectory of `${lib_dir}/lldb/clang`, that makes sense. I'm also not super knowledgeable about cmake.